mmv1/products/networkconnectivity/Spoke.yaml

```diff
@@ -44,6 +44,12 @@ examples:
       network_name: 'net'
       hub_name: 'hub1'
       spoke_name: 'spoke1'
+  - name: 'network_connectivity_spoke_linked_vpc_network_group'
+    primary_resource_id: 'primary'
+    vars:
+      network_name: 'net'
+      hub_name: 'hub1'
+      spoke_name: 'spoke1'
```

Comment:
```suggestion
      spoke_name: 'group-spoke1'
```

Reply:
This change is to add the prefix `tf-test-` to the name automatically. The prefix will help to sweep the leaked resources.